userspace/libraries/libgraphic/font/Parser.cpp

```diff
@@ -0,0 +1,8 @@
+#include <libgraphic/font/Parser.h>
+#include <libgraphic/font/TrueTypeFontFace.h>
+
+ResultOr<RefPtr<Graphic::Font::FontFace>> Graphic::Font::load(IO::Reader &reader)
+{
+    RefPtr<FontFace> result = TrueTypeFontFace::load(reader).unwrap();
```

Comment:
Why are you unwrapping it, you are losing the Result

Reply:
It couldn't convert from `ResultOr<RefPtr<Graphic::Font::TrueTypeFontFace>>` to  `ResultOr<RefPtr<Graphic::Font::FontFace>>`